docs/mkdocs.yml

```diff
@@ -34,33 +34,56 @@ site_dir: 'generated'
+              - Set Up Kubestellar Core Components: # common-subs/placeholder.md
+                - Try the "Common-setup": direct/common-setup-intro.md
+                - Step by Step: direct/common-setup-step-by-step.md
+                - (experimental) Via Helm Chart: direct/core-chart.md
+#                - Prepare the environment: common-subs/placeholder.md
+#                - Initialize Kubeflex: common-subs/placeholder.md
+#                - Install Kubestellar Core components: common-subs/placeholder.md
+#                - Create the Inventory & Transport Space: common-subs/placeholder.md
+#                - Create the Workload Description Space: common-subs/placeholder.md
+              - Set up and Register Workload Execution Clusters (WECs): direct/example-wecs.md
+#              - Register WECS with core: common-subs/placeholder.md
+              - Commission Workloads: direct/commisssion-workloads.md
+#          - Modifying Workloads: common-subs/placeholder.md
+          - Installation & Usage Examples:
+              - Example Cluster Preparation: direct/hosting-cluster.md
+              - Sample scenarios: direct/examples.md
+              - Deploy KubeStellar via Helm chart: direct/core-chart.md
+#              - Using existing hosting cluster: direct/hosting-cluster.md
+#              - KubeStellar on KIND: direct/ks-on-kind.md
+              - Deploying KubeStellar on K3D: direct/deploy-on-k3d.md
+#              - KubeStellar on OCP: direct/ks-on-ocp.md
           - Best practices: direct/best-practices.md
-          - Examples: direct/examples.md
+#          - API reference: common-subs/placeholder.md
+          - User Guide: direct/user-guide.md
   - Contributing: 
-      - Overview of contributing: direct/contributor.md
-      - Packaging: direct/packaging.md
-      - Release Process: direct/release.md
-      - Guidelines: Contribution guidelines/CONTRIBUTING.md
+      - How to join in: direct/contribute.md
+#      - Packaging: direct/packaging.md
+#      - Release Process: direct/release.md
+#      - Guidelines: Contribution guidelines/CONTRIBUTING.md
       - Code of Conduct: Contribution guidelines/coc.md
-      - License: Contribution guidelines/LICENSE.md
-      - Governance: Contribution guidelines/governance.md
-      - Onboarding: Contribution guidelines/onboarding.md
-      - Website: Contribution guidelines/operations/document-management.md
-      - Security: 
-          - Policy: Contribution guidelines/security/security.md
-          - Contacts: Contribution guidelines/security/security_contacts.md
+#      - License: Contribution guidelines/LICENSE.md
+#      - Governance: Contribution guidelines/governance.md
+#      - Onboarding: Contribution guidelines/onboarding.md
+#      - Website: Contribution guidelines/operations/document-management.md
+#      - Security:
+#          - Policy: Contribution guidelines/security/security.md
+#          - Contacts: Contribution guidelines/security/security_contacts.md
```

Comment:
This is governance stuff that has to stay visible on the website as part of being in the CNCF Sandbox.

Reply:
un-commented those lines and re-pushed the file.